resolwe/flow/models/secret.py

```diff
@@ -0,0 +1,75 @@
+"""Resolwe secrets model."""
+import uuid
+
+from fernet_fields import EncryptedTextField
+
+from django.conf import settings
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+
+class SecretManager(models.Manager):
+    """Manager for Secret objects."""
+
+    def create_secret(self, value, contributor, metadata=None, expires=None):
```

Comment:
Should this method always be used when creating secrets? If yes, then update docs and maybe also raise error when using the usual `.create(`. In this case it would also be nicer to return `handle, secret` to remove the need to query the secret by its handle if you intend to use the secret directly.

Alternatively if `.create` is allowed, then override it and call it in `create_secret`.

Reply:
Both ways are ok I guess, but prevent confusion.